### PR TITLE
composer scratchpad update

### PR DESCRIPTION
**Pull request by kdruckman**

### CODE REVIEW

**Comment by phenaproxima** [comment]
I just rolled a stable release of Diff (8.x-1.0), which includes the deprecation fixes through 8.7.x. So we can remove that from this PR :)

File: tests/packages.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+acquia/lightning:
```
Comment (phenaproxima) [inline tests/packages.yml:2]: @TravisCarden: This file is also referenced in .travis.yml as ORCA_PACKAGES_CONFIG_ALTER. Will these changes have any ill side effects due to that?

Comment (TravisCarden) [reply tests/packages.yml:2]: Actually, @phenaproxima, `tests/packages_alter.yml` is referenced by `ORCA_PACKAGES_CONFIG_ALTER`. Each file is used in only one case, so there should be problem.

Comment (phenaproxima) [reply tests/packages.yml:2]: Ah, thanks! I missed that, appreciate you pointing that out. Carry on!

**Pull request closed without merging**